src/core/midiDispatcher.cpp

```diff
@@ -64,9 +64,9 @@ std::mutex DT_mutex;
 
 //   -   -   -   -   -   -   -   -   -   -   -   -   -   -   -   -   -   -   -
 
-void _forward(const MidiMsg& mm, const std::string r){
+void forward_(const MidiMsg& mm, const std::string r){
 
-	u::log::print("[MDi::_forward] Forwarding message from %s to: %s...\n",
+	u::log::print("[MDi::forward_] Forwarding message from %s to: %s...\n",
 				mm.getMessageSender().c_str(), r.c_str());
```

Comment:
Since you are here :-)
```suggestion
				mm.getMessageSender(), r);
```

Reply:
Hm I guess I should wait with this until your pull request is merged, otherwise my code won't compile, don't you think?

Reply:
Oh right. I completely forgot about finishing https://github.com/monocasual/giada/pull/402
Thank you for the reminder!
Indeed it might compile, which proves that the C-style `printf` is very unsafe!